add job vacancy (#349)
open-life-science/open-life-science.github.io: _posts/2022-02-18-ols-is-hiring.md

---
layout: post
title: "Job vacancy: Community Researcher and Programme Coordinator - OLS is hiring!"
authors:
- bebatut
- emmyft
- malvikasharan
- yochannah
image: ../images/2022-02-18-for-hire.jpg
photos:
  name: Photo by Clem Onojeghuo on Unsplash
  url: https://unsplash.com/@clemono
---

OLS is hiring a Community Researcher and Programme Coordinator.

> _Note: If you like the scope of this job but unsure if your profile fits this role - please apply anyway. We are very happy to be contacted for general queries, please email jobs@openlifesci.org. You can also book a call with the Executive Director, Yo Yehudi, for specific questions about this vacancy: [book a 15 minute chat](https://calendly.com/yo-yehudi/ols-chat-with-yo?month=2022-02)._

Reviewer: ```suggestion
_Note: If you like the scope of this job but unsure if your profile fits this role - please apply anyway. We are very happy to be contacted for general queries, please email [jobs@openlifesci.org](mailto:jobs@openlifesci.org). You can also book a call with the Executive Director, Yo Yehudi, for specific questions about this vacancy: [book a 15 minute chat](https://calendly.com/yo-yehudi/ols-chat-with-yo?month=2022-02)._
```

> Open Life Science (OLS) is hiring an open, empathic, and creative colleague to coordinate  our cohort-based training and mentoring programmes and lead a research-based study to understand the long-term impacts of our work.

Reviewer 2: ```suggestion
Open Life Science (OLS) is hiring an open, empathic, and creative colleague to coordinate our cohort-based training and mentoring programmes and lead a research-based study to understand the long-term impacts of our work.
```
extra space

> Open Life Science (OLS, [https://openlifesci.org/](https://openlifesci.org/)) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. With the support of our community members, OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.

Reviewer: ```suggestion
Open Life Science (OLS, [{{ site.url }}]({{ site.url }})) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. With the support of our community members, OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.
```

> Open Life Science (OLS, [https://openlifesci.org/](https://openlifesci.org/)) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. With the support of our community members, OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.

Reviewer 2: ```suggestion
Open Life Science (OLS, [https://openlifesci.org/](https://openlifesci.org/)) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. With the support of our community members, OLS is committed to delivering high-quality training, mentoring and peer-based support to our community members.
```

> ## Position Purpose

Reviewer 2: ```suggestion
## Position purpose
```
No camel casing for consitency

> This role encompasses two areas of OLS’ work: community management and community research. As a community manager, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information in an open way. As a community researcher, you will help to carry out peer-reviewed research to understand transformational effects of open research training and mentoring interventions.

Reviewer 2: ```suggestion
This role encompasses two areas of OLS’ work: community management and community research. As a community manager, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information openly. As a community researcher, you will help to carry out peer-reviewed research to understand the transformational effects of open research training and mentoring interventions.
```

## Key responsibilities
- Community management and operational tasks involve communicating with the community members via Slack space, mailing lists and social media.
- Lead a research-based study to assess the long-term impact of OLS’s programme on graduates and their communities.
- 5-10% of the time for this position will be dedicated to supporting other open projects. This will include working in collaboration with community members, contributing to other open science projects on behalf of OLS and representing OLS in open science community events.
- Implement, improve, and iterate on research practises for yourself and the OLS team, including ethics in research, version control, and open communications.

> ## Essential selection criteria:

Reviewer: ```suggestion
## Essential selection criteria

```

Author: this shows as outdated so I can't merge it for some reason, will do manually.

- Ability to learn high-level complex concepts quickly
- Dedicated to open, trusting, and collaborative work; and understanding the importance of communication for that kind of work environment.

The position is funded for 18 months to carry out the activities as described in this grant from the Wellcome Trust’s Open Research Fund: [https://doi.org/10.5281/zenodo.5267933](https://doi.org/10.5281/zenodo.5267933). We are happy to consider job share applications or part-time applications. Our base contract assumes around 35 hours a week as a full time role, with option to start/end and work at times that are flexible and work for you. You will work remotely for 17.5 (minimum part time) to 35 hours per week (if full time), and the annual salary range is £30,000 to £40,000 with benefits, pro-rata for part time roles (equivalent to £16-£22 per hour).

OLS is committed to providing a nurturing and flexible work environment. This job will be fully remote, and we are happy to consider applicants from any country. Ideally 2 hours or more of your work day should overlap with the OLS leadership team based in the UK and Europe (UTC to UTC+2).

> ## How to apply

Reviewer: ```suggestion
## How to apply

```

> Please send a CV (1-4 pages) and cover letter (no more than 800 words) stating why this opportunity is compelling to you, what you want to achieve in this space, and what you bring to this work to: jobs@openlifesci.org.

Reviewer: ```suggestion
Please send a CV (1-4 pages) and cover letter (no more than 800 words) stating why this opportunity is compelling to you, what you want to achieve in this space, and what you bring to this work to: [jobs@openlifesci.org](mailto:jobs@openlifesci.org).
```

---
layout: post
title: "Job vacancy: Community Researcher and Programme Coordinator - OLS is hiring!"
authors:
- bebatut
- emmyft
- malvikasharan
- yochannah
image: ../images/2022-02-18-for-hire.jpg
photos:
  name: Photo by Clem Onojeghuo on Unsplash
  url: https://unsplash.com/@clemono
---

OLS is hiring a **Community Researcher and Programme Coordinator**.

_Note: If you like the scope of this job but unsure if your profile fits this role - please apply anyway. We are very happy to be contacted for general queries, please email [jobs@openlifesci.org](mailto:jobs@openlifesci.org). You can also book a call with the Executive Director, Yo Yehudi, for specific questions about this vacancy: [book a 15 minute chat](https://calendly.com/yo-yehudi/ols-chat-with-yo?month=2022-02)._

Open Life Science (OLS) is hiring an open, empathic, and creative colleague to coordinate our cohort-based training and mentoring programmes and lead a research-based study to understand the long-term impacts of our work.

> Open Life Science (OLS, [{{ site.url }}]({{ site.url }})) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.

Author: ```suggestion
Open Life Science (OLS, [{{ site.url }}]({{ site.url }})) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. OLS is committed to delivering high-quality training, mentoring and peer-based support to our community members.
```

> Open Life Science (OLS, [https://openlifesci.org/](https://openlifesci.org/)) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.

Author: ```suggestion
Open Life Science (OLS, [{{ site.url }}]({{ site.url }})) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.
```

> Open Life Science (OLS, [https://openlifesci.org/](https://openlifesci.org/)) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.

Author: ```suggestion
Open Life Science (OLS, [{{ site.url }}]({{ site.url }})) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.
```

## Position purpose

> This role encompasses two areas of OLS’ work: community research and programme coordination. As a community researcher, you will help to carry out peer-reviewed research to understand transformational effects of open research training and mentoring interventions. As a programme coordinator, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information openly. 

Author: ```suggestion
This role encompasses two areas of OLS' work: community research and programme coordination. As a community researcher, you will help to carry out peer-reviewed research to understand transformational effects of open research training and mentoring interventions. As a programme coordinator, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information openly. 
```

Author: smart quote causing havoc

> This role encompasses two areas of OLS' work: community research and programme coordination. As a community researcher, you will help to carry out peer-reviewed research to understand transformational effects of open research training and mentoring interventions. As a programme coordinator, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information openly. 

Author: ```suggestion
This role encompasses two areas of OLS' work: community management and community research. As a community manager, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information openly. As a community researcher, you will help to carry out peer-reviewed research to understand the transformational effects of open research training and mentoring interventions.

Author: @malvikasharan I had to do some manual conflict resolutions between the two sets of edits - can you check if this is the final language you wanted?
